components/Filters/RadioContainsAttachements.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { Radio, Row } from 'react-components';
+import { noop } from 'proton-shared/lib/helpers/function';
+
+function RadioContainsAttachements({ comparator, onChange }) {
+    const [value, setValue] = useState(comparator);
```

Comment:
Do you really need a `state` here since you have one also [outside](https://github.com/ProtonMail/react-components/pull/75/files#diff-881f831d03e0a620e6d31f1f4980efb2R113)?